src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private final ColorSupplier colorSupplier = new ColorSupplier();
+    private final Random random = new Random();
+
+    public Ball getRandomBall() {
+        String color = colorSupplier.getRandomColor().toString();

```

Comment:
When you start to store a `Color`enum value in `Ball` class it will be redundant to call `toString` method and your will be able to simplify your code in that way`Color color = colorSupplier.getRandomColor()`.
Isn't that wonderful?